### ISSUE TITLE
Add Dino Depot support

### ISSUE THREAD

**Issue by axieum** [miragedmuk/ASV#34]
Dino Depot is a popular mod that replaces Cryopods, and as such searching tamed creatures on the map is not possible with ASV.

**Comment by miragedmuk** [comment]
Ark Evolved or Ascended?

I run Dino Depot on my Ascended server and this seems to supported by ASV?

If you have a specific save that doesn't load or parse correctly please upload it somewhere I can download a copy to investigate.

**Comment by axieum** [comment]
Indeed, it is miraculously working now! Thanks again for this program, it's so good.
